probatus/feature_elimination/feature_elimination.py

```diff
@@ -649,12 +662,12 @@ def fit_compute(self, X, y, sample_weight=None, columns_to_keep=None, column_nam
         )
         return self.compute()
 
-    def get_reduced_features_set(self, num_features):
+    def get_reduced_features_set(self, num_features=None):
         """
         Gets the features set after the feature elimination process, for a given number of features.
 
         Args:
-            num_features (int):
+            num_features (int, optional):
```

Comment:
Can you add to docstring that if None, then it uses the formula to calculate the best one?